prov/efa/src/rxr/rxr_init.c

```diff
@@ -83,12 +83,31 @@ struct rxr_env rxr_env = {
 	.rnr_retry = EFA_RNR_INFINITE_RETRY,
 };
 
+
+static int efa_convert_mem_monitor_str(const char *value)
+{
+        int i;
+
+        if (!value) {
+                return EFA_UFFD_MONITOR;
+	}
+
+        for (i = 0; efa_mem_monitors[i]; i++) {
+                if (!strcasecmp(value, efa_mem_monitors[i]))
+                        return i;
+        }
+        return EFA_UFFD_MONITOR;
+}
+
 /* @brief Read and store the FI_EFA_* environment variables.
  */
 static void rxr_init_env(void)
 {
+	char *system_mem_monitor_str = NULL;
 	int fork_safe = 0;
 
+	fi_param_get_str(&rxr_prov, "default_system_mem_monitor", &system_mem_monitor_str);
```

Comment:
I think we need to check return value of `fi_param_get_str` to determine whether the environment variable is set.